etc/HIProdOfflineConfiguration.py

```diff
@@ -1153,11 +1166,61 @@
                write_dqm=False,
                write_aod=False,
                archival_node=None,
-               siteWhitelist = ["T2_CH_CERN_P5", "T2_CH_CERN"],
                tape_node="T0_CH_CERN_MSS",
                disk_node="T2_CH_CERN",
                scenario=ppScenario)
 
+## DAQ TRANSFER TEST PDs (fall 2024)
+DATASETS_DAQ_TFTEST = ["TestHLTPhysics0","TestHLTPhysics1", "TestHLTPhysics2", "TestHLTPhysics3",
+            "TestHLTPhysics4", "TestHLTPhysics5", "TestHLTPhysics6","TestHLTPhysics7",
+            "TestHLTPhysics8", "TestHLTPhysics9","TestHLTPhysics10","TestHLTPhysics11",
+            "TestHLTPhysics12", "TestHLTPhysics13","TestHLTPhysics14","TestHLTPhysics15",
+            "TestHLTPhysics16", "TestHLTPhysics17","TestHLTPhysics18","TestHLTPhysics19",
+            "TestHLTPhysics20", "TestHLTPhysics21","TestHLTPhysics22","TestHLTPhysics23",
+            "TestHLTPhysics24", "TestHLTPhysics25","TestHLTPhysics26","TestHLTPhysics27",
+            "TestHLTPhysics28", "TestHLTPhysics29","TestHLTPhysics30","TestHLTPhysics31",
+            "TestHLTPhysics32", "TestHLTPhysics33","TestHLTPhysics34","TestHLTPhysics35",
+            "TestHLTPhysics36", "TestHLTPhysics37","TestHLTPhysics38","TestHLTPhysics39"]
+
+STREAMS_DAQ_TFTEST = DATASETS_DAQ_TFTEST
```

Comment:
Fwiw, this (`STREAMS_DAQ_TFTEST`) and L1211 (`STREAMS_DAQ_TFTEST_ppRef`) are entirely accurate, because in this the number of streams is half of the number of PDs.

The streams are
```
TestHLTPhysics[0-19]
TestHLTPhysicsA[0-4]
TestHLTPhysicsB[0-4]
TestHLTPhysicsC[0-4]
TestHLTPhysicsD[0-4]
```

Reply:
That's ok from our side. As long as all streams are included in the list, it works.